docker-robottests-targets/py38_no_python_symlink/Dockerfile

```diff
@@ -0,0 +1,4 @@
+# Copyright (C) 2019, Nokia
+
+FROM py3-image:latest
```

Comment:
Better to use Docker Hub image rather than local as the existence of the local depends on the build order.

Reply:
Refactored